packages/common/src/components/DeactivatedAccount.tsx

```diff
@@ -0,0 +1,116 @@
+import React from "react";
+import { useTranslation } from "next-i18next";
+import styled from "styled-components";
+import IconButton from "./IconButton";
+import { IconArrowRight } from "hds-react";
+import { fontBold, fontRegular } from "../common/typography";
+
+const ErrorContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  max-width: var(--container-width-xl);
+  margin: 0 auto;
+  gap: 42px;
+  padding-block: 42px;
+  padding-inline: var(--spacing-s);
+  --spacing-hz: var(--spacing-s);
+  img {
+    width: 185px;
```

Comment:
magic values, why?

Reply:
As many as possible of the magic values removed, all of the remaining ones pertain to the image dimensions.

Reply:
Also, switched the image dimensions into their own `const`s - to lessen the "magic" component :D